modules/widgets/src/zoom-widget.tsx

```diff
@@ -0,0 +1,114 @@
+import {FlyToInterpolator} from '@deck.gl/core';
+import type {Deck, Viewport, Widget, WidgetPlacement} from '@deck.gl/core';
+import {h, render} from 'preact';
+
+interface ZoomWidgetProps {
+  id: string;
+  viewId?: string | null;
+  placement?: WidgetPlacement;
+  zoomInLabel?: string;
+  zoomOutLabel?: string;
+  transitionDuration?: number;
+  style?: Partial<CSSStyleDeclaration>;
+  className?: string;
+}
+
+class ZoomWidget implements Widget<ZoomWidgetProps> {
+  id = 'zoom';
+  props: ZoomWidgetProps;
+  placement: WidgetPlacement = 'top-left';
+  viewId = null;
+  viewport: Viewport;
+  deck?: Deck;
+  element?: HTMLDivElement;
+
+  constructor(props: ZoomWidgetProps) {
+    this.id = props.id || 'zoom';
+    this.viewId = props.viewId || null;
+    this.placement = props.placement || 'top-left';
+    props.transitionDuration = props.transitionDuration || 200;
+    props.zoomInLabel = props.zoomInLabel || 'Zoom In';
+    props.zoomOutLabel = props.zoomOutLabel || 'Zoom Out';
+    props.style = props.style || {};
+    this.props = props;
+  }
+
+  onAdd({deck}: {deck: Deck}): HTMLDivElement {
+    const {style, className} = this.props;
+    const element = document.createElement('div');
+    element.classList.add('deckgl-widget', 'deckgl-widget-zoom');
+    if (className) element.classList.add(className);
+    Object.entries(style).map(([key, value]) => element.style.setProperty(key, value as string));
+    const ui = (
+      <div className="deckgl-widget-button-group">
```

Comment:
Can you please adjust this to have a 1px gap between the buttons? I think its fine to add new css classes for the group use case similar to `.deckgl-widget-button-border` since they can reuse the same variables where it makes sense, such as `--button-shadow`